completions/bash/multipass

```diff
@@ -19,7 +19,7 @@ _multipass_complete()
         local state=$1
 
         local cmd="multipass list --format=csv --no-ipv4"
-        [ -n "$state" ] && cmd="$cmd | \grep -E '$state'"
+        [ -n "$state" ] && cmd="$cmd | \awk -F',' '\$2 == \"$state\"'"
```

Comment:
Good improvement. This is to avoid falling pray to instances called "Running-Rabbit", correct?

Is there any special need for this with snapshots? If not, could you cover it in a drive-by PR (the same as the other one would be fine).